### ISSUE TITLE
No longer working?

### ISSUE THREAD

**Issue by Aida-Enna** [Lerofni/TooltipNotes#21]
Just tested this post 6.4 and the notes no longer show on item descriptions. Clicking Add/Edit note also does nothing on HQ items and in some cases with NQ items opens the "Search Recipes Using This Material" UI. It's still detecting notes though - If I add a note using /tnnote it changes "Add Note" to "Edit Note" but it still doesn't show the note or let me edit it.

**Comment by Lerofni** [comment]
Yup, notes aren't being displayed because currently the sig in Xivcommon is not updated yet, the context menus are broken because I haven't updated Dalamud.Contextmenu yet because just updating one of the 2 dependencies doesn't make sense. I may look into what is it would take to either switch to Criticals lib or my own sigs for the tooltips part.

**Comment by Aida-Enna** [comment]
Oooh, makes sense! Take your time, just wanted to make sure I wasn't breaking something. Thank you! 👍
